components/server/src/workspace/workspace-starter.ts

```diff
@@ -1746,4 +1752,34 @@ export class WorkspaceStarter {
 
         return result;
     }
+
+    /**
+     * This method is temporary until we moved image-builder into workspace clusters
+     * @param user
+     * @param workspace
+     * @param instance
+     * @returns
+     */
+    protected async getImageBuilderClient(user: User, workspace: Workspace, instance?: WorkspaceInstance) {
```

Comment:
This is the same/similar as https://github.com/gitpod-io/gitpod/pull/11103/files#diff-d6fdb9bb15400f4fbba00db81b2ce6cb7f7bb566dd97b93b9fd4e3fc9a79f479R3121, can we unify?